auth/jwt/middleware_test.go

```diff
@@ -0,0 +1,105 @@
+package jwt
+
+import (
+	"testing"
+
+	jwt "github.com/dgrijalva/jwt-go"
+
+	"golang.org/x/net/context"
+)
+
+var (
+	kid           = "kid"
+	key           = []byte("test_signing_key")
+	method        = jwt.SigningMethodHS256
+	invalidMethod = jwt.SigningMethodRS256
+	claims        = Claims{"user": "go-kit"}
+	signedKey     = "eyJhbGciOiJIUzI1NiIsImtpZCI6ImtpZCIsInR5cCI6IkpXVCJ9.eyJ1c2VyIjoiZ28ta2l0In0.14M2VmYyApdSlV_LZ88ajjwuaLeIFplB8JpyNy0A19E"
+	invalidKey    = "eyJhbGciOiJSUzI1NiIsInR5cCI6IkpXVCJ9.e30.vKVCKto-Wn6rgz3vBdaZaCBGfCBDTXOENSo_X2Gq7qA"
```

Comment:
Where did these strings come from? Can you add a comment indicating how to generate them?